Utils/Hash/openssl/sha.h

```diff
@@ -4,7 +4,7 @@
  *
  * This package is an SSL implementation written
  * by Eric Young (eay@cryptsoft.com).
- * The implementation was written so as to conform with Netscapes SSL.
+ * The implementation was written so as to conform with Netscape's SSL.
```

Comment:
This is probably a third party file. I'm happy to exclude the file.

Reply:
Let's keep it.  It is third party, but I think it is ok.